commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -1,62 +1,64 @@
-@import common.commercial.HostedVideoPage
+@import common.commercial.hosted.HostedVideoPage
 @(page: HostedVideoPage)(implicit request: RequestHeader)
 @import views.html.hosted.guardianHostedHeader
 
 @main(page, Some("commercial")) { } {
-    @guardianHostedHeader("hosted-video-page", page.logoUrl)
+    @guardianHostedHeader("hosted-video-page", page.campaign.logo.url)
     <div class="l-side-margins hosted__side hosted-video-page">
         <section class="hosted-tone--dark">
             <div class="adverts hosted__container--full">
                 <div class="u-responsive-ratio u-responsive-ratio--hd">
                     <video
-                    data-duration="@page.video.duration"
-                    data-media-id="@page.video.mediaId"
+                    data-duration="@{page.video.duration}"
+                    data-media-id="@{page.video.mediaId}"
                     controls="controls"
                     preload="auto"
                     style="width: 100%;"
                     data-block-video-ads="true"
-                    poster="@page.video.posterUrl"
+                    poster="@{page.video.posterUrl}"
                     class="vjs-hosted__video hosted__video gu-media--video vjs vjs-tech-html5 vjs-paused vjs-controls-enabled vjs_video_1-dimensions vjs-user-active">
-                        <source type="video/mp4" src="@page.video.srcUrlMp4">
-                        <source type="video/webm" src="@page.video.srcUrlWebm">
-                        <source type="video/ogg" src="@page.video.srcUrlOgg">
-                        <source type="video/m3u8" src="@page.video.srcM3u8">
+                        <source type="video/mp4" src="@{page.video.srcUrlMp4}">
+                        <source type="video/webm" src="@{page.video.srcUrlWebm}">
+                        <source type="video/ogg" src="@{page.video.srcUrlOgg}">
+                        <source type="video/m3u8" src="@{page.video.srcM3u8}">
                     </video>
                     <div class="hosted-fading js-hosted-fading">
                         <div class="hosted__video-overlay"></div>
                         <div class="hosted__meta">
-                            <h1 class="hosted__heading hosted-tone--renault">@page.video.title</h1>
+                            <h1 class="hosted__heading hosted-tone--@{page.campaign.owner.toLowerCase}">@{page.video.title}</h1>
```

Comment:
Can `page.campaign.owner` include spaces? If so, might want to replace them with empty string to ensure a single class is added.